src/components/common/TextContent.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+/**
+ * TODO: Update component to use JSON or Markdown file to render text content
+ * 
+ * Required Props:
+ * contentTitleString {String} – String representing the title a sub module
+ * 
+ * Optional props:
+ * contentFirstParagraph {String} – String representing the first paragraph of content of a sub module
```

Comment:
I'd make this an array of Paragraph strings. Otherwise we are maxed at 3 and going to have empty paragraph tags if they are null (which adds spacing)

Basically replace this:
`      <p>{contentFirstParagraph}</p>
      <p>{contentSecondParagraph}</p>
      <p>{contentThirdParagraph}</p>`

With something like this:
`
{contentParagraphs.map((paragraph, index) => <p key={index}>{paragraph}</p>}
}
`
map will go through the array of strings called contentParagraphs and return a list of paragraph tags. React is smart enough to know how to render a list of elements. The key thing is just a fancy react thing you have to do so it can keep track of which is which.

Reply:
Alternatively:
You can add a prop called `children`. This is a special name react uses and the prop type needs to be a React node. Then you can use it like children in html.

For example:
```
<TextContent><p>my content</p></TextContent>
```
```
function textContent(props) {
   const {children} = props;
   return (
      <div>
          <h1>Title</h1>
          {children}
      </div>
   );
}
```

Although since we want to autogenerate these from from JSON the above approach might be more scalable.

Reply:
Thank you! Fixed.